ios/brave-ios/Sources/PlaylistUI/PlayerModel.swift

```diff
@@ -60,7 +59,9 @@ public final class PlayerModel: ObservableObject {
       log.warning("PlayerModel deallocated without first stopping the underlying media.")
       stop()
     }
-    try? AVAudioSession.sharedInstance().setActive(false)
+    DispatchQueue.global().async {
+      try? AVAudioSession.sharedInstance().setActive(false)
```

Comment:
hmm why wrapped it under `DispatchQueue.global().async` ? Curious cause we have audio session deactivation is in multiple place in app.

Reply:
The goal was to ensure everything related to AVKit/AVFoundation APIs were called off main because there are known bugs in those frameworks that cause lock ups on main